Sources/PenguinParallel/NonblockingThreadPool/NumberOperations.swift

```diff
@@ -31,7 +31,7 @@ internal func makeCoprimes(upTo n: Int) -> [Int] {
   return coprimes
 }
 
-/// Reduce `lhs` into `[0, size)`.
+/// Returns a value deterministically selected from `0..<size`.
```

Comment:
I worked pretty hard on my proposed replacement summary.  This one doesn't even explain what `lhs` is.  Oh well.